projects/extension/build.py

```diff
@@ -507,14 +497,6 @@ def prior_versions() -> list[str]:
     return versions()[1:] if len(versions()) > 1 else []
 
 
-def deprecated_versions() -> set[str]:
-    return {
-        "0.3.0",  # deprecated
-        "0.2.0",  # deprecated
-        "0.1.0",  # deprecated
-    }
-
-
```

Comment:
can we leave this and return an empty set? then we're set up if we need to deprecate versions in the future

Reply:
yes! done.